patches/corefx/0006-Do-not-fail-on-test-chmod.patch

```diff
@@ -0,0 +1,25 @@
+From ef90f9473a5b2b24d9c1a523aefa7d6478bd3335 Mon Sep 17 00:00:00 2001
+From: Tomas Weinfurt <tweinfurt@yahoo.com>
+Date: Tue, 12 Nov 2019 13:24:37 -0800
+Subject: [PATCH] do not fail on test chmod
+
+---
+ external/runtime/runtime.depproj | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/external/runtime/runtime.depproj b/external/runtime/runtime.depproj
+index 3ce0ed89bc..1298019efd 100644
+--- a/external/runtime/runtime.depproj
++++ b/external/runtime/runtime.depproj
+@@ -60,7 +60,7 @@
+           SkipUnchangedFiles="true"
+           UseHardlinksIfPossible="$(UseHardlink)" />
+ 
+-    <Exec Command="chmod +x $(TestHostRootPath)%(DotnetExe.Filename)%(DotnetExe.Extension)" Condition="'$(OS)' != 'Windows_NT'"/>
++    <Exec Command="chmod +x $(TestHostRootPath)%(DotnetExe.Filename)%(DotnetExe.Extension) || true" Condition="'$(OS)' != 'Windows_NT'"/>
```

Comment:
Has someone from corefx looked at this change? We are (or were) hitting this on Fedora 32 and Arch Linux as well: https://github.com/dotnet/source-build/issues/1310#issuecomment-543163427. Maybe this should be merged into corefx if it's the correct general fix.

Reply:
yes, that is my plan. but I'll do it after repo merge. I did testing and we don't need it at all any more. The dotnet is executable as it should on its own.

Reply:
Does the build work without this change?  I wouldn't expect test code to be getting hit in our normal build.

Reply:
@crummel The build fails here for Fedora 32 and Arch Linux too: https://github.com/dotnet/source-build/issues/1310#issuecomment-543163427. This code path is definitely invoked in source-build.

Reply:
no, it does not build @crummel 
It seems like it got moved to Arcade in master. 
I'll take a look after repo merge is over. 
cc: @safern

Reply:
BTW I think this is old time paranoia when we did not know how to set and preserve executable bits on Unix.